config.yaml

```diff
@@ -0,0 +1,420 @@
+    name: ssl_certificate_v1.py
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/httphealthcheck_v1/httphealthcheck_v1.py
+    name: httphealthcheck_v1.py
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/httploadbalancer_v1/httploadbalancer_v1.py
+    name: httploadbalancer_v1.py
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/backendservice_v1/backendservice_v1.py
+    name: backendservice_v1.py
+  - path: star-binnys-com-ca.pem
+  - path: star-binnys-com-key.pem
+  - path: bucket_v1.py
+  - path: cloudsql_instance_v1.py
+  - path: firewall_rule_v1.py
+
+##############################################################################
+  
+  resources:
+    - type: network_v1
+      name: first-deployment-network
+      properties:
+        autoCreateSubnetworks: False
+        network_description: 'first-deployment-Shared Network'
+
+    - type: subnetwork_v1
+      name: &Production_Web
+      properties:
+        network: $(ref.first-deployment-network.selfLink)
+        ipCidrRange: &NETWORK_PROD_WEB 192.168.1.0/24
+        region: &REGION us-central1
+   
+    - type: subnetwork_v1
+      name: &Production_Data 
+      properties:
+        network: $(ref.first-deployment-network.selfLink)
+        ipCidrRange: &NETWORK_PROD_DATA 192.168.2.0/24
+        region: *REGION 
+
+    - type: subnetwork_v1
+      name: &Production_Shared
+      properties:
+        network: $(ref.first-deployment-network.selfLink)
+        ipCidrRange: &NETWORK_PROD_SHARED 192.168.3.0/24
+        region: *REGION 
+
+    - type: subnetwork_v1
+      name: &Staging
+      properties:
+        network: $(ref.first-deployment-network.selfLink)
+        ipCidrRange: &NETWORK_STAGING 192.168.4.0/24
+        region: *REGION    
+
+    - type: subnetwork_v1
+      name: &Development
+      properties:
+        network: $(ref.first-deployment-network.selfLink)
+        ipCidrRange: &NETWORK_DEV 192.168.5.0/24
+        region: *REGION     
+
+#Defining IGM 
+  - type: gce_instancetemplate_v1.py
+    name: Magento 
+    properties:
+      machineType: n1-standard-4
+      image_url: "projects/centos-cloud/global/images/family/centos-7"
+      networkInterfaces:
+        - accessConfigs:
+            - {name: external-nat, type: ONE_TO_ONE_NAT}
+      disks:
+        - deviceName: 'boot'
+        - boot: true
+        - autoDelete: false
+
+    - type: igm_v1
+      name: Mgento
+      properties:
+        region: *REGION
+        targetSize: 1
+        instanceTemplate:
+
+#Autoscale group for MIG  
+    - type: autoscaler_v1.py
+      name: IGM-Magento
+      properties:
+        region: *Region
+        autoscalingPolicy.minNumReplicas: 1
+        autoscalingPolicy.maxNumReplicas: 4
+        target: $(ref.Magento.safeLink)
+
+#Load balancer
+    - type: httploadbalancer_v1.py
+      name: httploadbalancer
+      backendProperties:
+        backends:
+          - group: $(ref.IGM-Magento.safeLink)
```

Comment:
You must reference an IGM resource not an Autoscaler.

Reply:
Updated